internal/commands/prepare/prepare.go

```diff
@@ -92,6 +95,15 @@ func runPrepareVM(cmd *cobra.Command, args []string) error {
 		return err
 	}
 
+	if gitLabEnv.JobImage == "" {
+		if defaultImage == "" {
+			return fmt.Errorf("%w: CUSTOM_ENV_CI_JOB_ID is missing and no default image was provided", ErrFailed)
```

Comment:
Improve discoverability:

```suggestion
			return fmt.Errorf("%w: CUSTOM_ENV_CI_JOB_ID is missing and no --default-image was set", ErrFailed)
```